ACM_General/events/models.py

```diff
@@ -3,92 +3,193 @@
 from django.utils.translation import ugettext_lazy as _
 from . import managers
 import uuid
+from django.core.exceptions import ValidationError
+from django.conf import settings
 
-##
-# To stop circular import errors and allow for djangos model resolution
-# to do its thing
-##
 
+# To stop circular import errors and allow for djangos model resolution to
+# work as it should.
 User = 'accounts.User'
 SIG = 'sigs.SIG'
 
-# Create your models here.
+def get_path_for_flier(instance, filename):
+    """
+    Used to obtain a path that uses an instance of the Event object to get the "date_hosted" variable.
```

Comment:
Used to obtain a path based on an instance of the Event (try to make this a cross reference if you can) object pasted into the function.